### ISSUE TITLE
About the mel rvq training question

### ISSUE THREAD

**Issue by ZZY2019-del** [tencent-ailab/MuQ#11]
Hello, may I ask how many steps are appropriate for training mel rvq, what is the final converged loss value, and should we mainly focus on commitment_loss or recon_loss?

**Comment by juhayna-zh** [comment]
Have you checked the released MelRVQ and SSL training code in https://github.com/tencent-ailab/MuQ/tree/main/src/recipes/pretrain#step2-prepare-the-mel-rvq? There are MelRVQ config and checkpoints for your reference.

**Comment by ZZY2019-del** [comment]
Yes, I observed that mel-rvq was trained for 4000 steps maybe in an hour, and I want to know if its loss has already converged. Are these steps of training sufficient for the after MUQ training?

> Have you checked the released MelRVQ and SSL training code in https://github.com/tencent-ailab/MuQ/tree/main/src/recipes/pretrain#step2-prepare-the-mel-rvq? There are MelRVQ config and checkpoints for your reference.
